blockchain/api.go

```diff
@@ -1,5 +1,290 @@
-//go:generate go run utils/generate_api.go
-
 package blockchain
 
-// nothing here yet
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/sonm-io/core/blockchain/tsc"
+	"github.com/sonm-io/core/blockchain/tsc/api"
+	"github.com/sonm-io/core/blockchain/utils"
+	"math/big"
+)
+
+const ethEndpoint string = "https://rinkeby.infura.io/00iTrs5PIy0uGODwcsrb"
+
+var ethClient *ethclient.Client = nil
```

Comment:
Store `ethClient` as part of `BlockchainAPI` struct, no reason to keep it as the variable into the package.